data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -652,6 +655,27 @@
             animate=yes
         [/kill]
 
+        [if]
+            [not]
+                [have_unit]
+                    id="Sir Gerrick"
+                [/have_unit]
+            [/not]
+            [then]
+                {SIR_GERRICK}
```

Comment:
Personally, I think the goals should mean exactly what they say.

In this case, if you manage to kill the bandit leader, fine. But you still need to wander to the fort and find Sir Gerrick.

But I was going for least change, and simply putting him in was what was needed for Debug/Next so that was good enough for non-debug as a first-pass.